### PR TITLE
LPS-141905 style book preview in staging

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-141905

### CODE REVIEW

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/EditStyleBookEntryDisplayContext.java [hunk 1]
```diff
@@ -312,8 +318,7 @@ private JSONObject _getPageOptionJSONObject() {
 				PortletURL itemSelectorURL = _itemSelector.getItemSelectorURL(
 					RequestBackedPortletURLFactoryUtil.create(
 						_httpServletRequest),
-					_themeDisplay.getScopeGroup(),
-					_themeDisplay.getScopeGroupId(),
+					group, group.getGroupId(),
```
Comment (ealonso) [inline modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/EditStyleBookEntryDisplayContext.java:321]: @lfbesada 
Why we need group and refererGroupId for the itemSelectorURL

File: modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/EditStyleBookEntryDisplayContext.java [hunk 2]
```diff
@@ -286,8 +291,9 @@ private JSONObject _getOptionJSONObject(int layoutType) {
 	}
 
 	private JSONObject _getPageOptionJSONObject() {
-		int total = LayoutLocalServiceUtil.getLayoutsCount(
-			_themeDisplay.getScopeGroupId());
+		Group group = _getPreviewItemsGroup();
+
+		int total = LayoutLocalServiceUtil.getLayoutsCount(group.getGroupId());
 
 		int numItems = 4;
```
Comment (ealonso) [inline modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/EditStyleBookEntryDisplayContext.java:298]: @lfbesada 
What do you think about using?

`int numItems = Math.min(total, 4);`

**Comment by lfbesada** [comment]
Hi @ealonso 

Thank you for the tips! I am going to close this PR and open a new one with the fixes.

Thanks!

**Comment by lfbesada** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/6333

**Pull request closed without merging**